documentation/man/features/deploy.rst

```diff
@@ -0,0 +1,105 @@
+=========
+kw-deploy
+=========
+
+.. _deploy-doc:
+
+-----------------
+deploy the kernel
+-----------------
+
+SYNOPSIS
+========
+*kw* (*d* | *deploy*) [\--remote [<remote>:<port>] | \--local | \--vm]
+                      [-r | \--reboot] [-m | \--modules] [-s | \--ls-line]
+                      [-l | \--list] [(-u | \--uninstall) <kernel-name>[,...]]
+
```

Comment:
Why are we using this pattern here? I prefer the one that you use in the configm.

Reply:
So the idea is to show the different ways to use the commands, and in the case of `configm` the options are not supposed to used at the same time, and here it is. I agree it looks kind of messy, and looking at some of the man pages for git it sometimes uses the `git <feature> [<options>]` pattern to simplify things, I'm thinking we should probably use it for some of our commands as well.